### ISSUE TITLE
Bug Report: Crash on large selection

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#1246]
Separating #950 into 2 issues. #950 was a "capture" issue. This is a "copy the selection" issue.

# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18917.1000]
```

# Steps to reproduce

1. Maximize window
2. Make a selection from top-left corner to bottom-right
3. Right-click to copy

# Expected behavior
Obviously, to not crash. 

# Actual behavior
Crash (presumably because copy is too big.)

**Comment by carlos-zamora** [comment]
More Info:

So, I'm debugging this and I notice that the issue arises in `TermControl::CopySelectionToClipboard()`:
```c++
const auto copiedData = _terminal->RetrieveSelectedTextFromBuffer(trimTrailingWhitespace);
_terminal->ClearSelection();
```
If I set a breakpoint on the first line, everything works fine.
If I set a breakpoint on the second line, it crashes.

**Comment by carlos-zamora** [comment]
Turns out that this is the fixed by #1254 and is a dup of #1165. So nevermind!
